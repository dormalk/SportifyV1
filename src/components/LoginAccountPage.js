import React from 'react';
import LoginAccountBox from './LoginAccountBox';

export const LoginAccountPage = () => (
    <div className="login-page">
        <LoginAccountBox/>
    </div>
);

export default LoginAccountPage; 

