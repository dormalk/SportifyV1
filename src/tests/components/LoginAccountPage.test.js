
import React from 'react';
import { shallow } from 'enzyme';
import LoginAccountPage from '../../components/LoginAccountPage';

test('should render LoginAccountPage correctly', () => {
    const wrapper = shallow(<LoginAccountPage/>);
    expect(wrapper).toMatchSnapshot();
});