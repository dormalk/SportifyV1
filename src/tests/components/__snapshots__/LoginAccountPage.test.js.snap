// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginAccountPage correctly 1`] = `
<div
  className="login-page"
>
  <Connect(LoginAccountBox) />
</div>
`;
